### ISSUE TITLE
traffic_constraints ignored

### ISSUE THREAD

**Issue by anna-braun** [JuPedSim/jpscore#564]
When testing the schedule function I wanted to "temp_close" all doors with by traffic_constraints. But this does not work, the traffic_constraints were ignored. Using an event file to "temp_close" the doors works. But it's a bug anyway.

 [Archiv.zip](https://github.com/JuPedSim/jpscore/files/3879130/Archiv.zip)

**Comment by schroedtert** [comment]
There is the `doors` tag missing, traffic_constraints should be like:
```
    <traffic_constraints>
      <doors>
        <door trans_id="0" caption="NaN" state="temp_close" />
        <door trans_id="1" caption="NaN" state="temp_close" />
        <door trans_id="2" caption="NaN" state="temp_close" />
        <door trans_id="3" caption="NaN" state="temp_close" />
        <door trans_id="4" caption="NaN" state="temp_close" />
        <door trans_id="5" caption="NaN" state="temp_close" />
        <door trans_id="6" caption="NaN" state="temp_close" />
        <door trans_id="7" caption="NaN" state="temp_close" />
        <door trans_id="8" caption="NaN" state="temp_close" />
        <door trans_id="9" caption="NaN" state="temp_close" />
      </doors>
    </traffic_constraints>
```

**Comment by anna-braun** [comment]
Thank you! This was a stupid mistake..
But there is an other problem. I added the doors tag, but now the simulation stops at t=0 sec, because there is no open door. This Problem was solved for the (event) door statuses (#533). Is there a difference between events and traffic_constraints?

**Comment by anna-braun** [comment]
Works with PR #565
